### DISCUSSION TITLE
Won't display?

### DISCUSSION THREAD

**Discussion by Thatcrazytaco222** [ppy/osu#22862] [Q&A]
i've tried numerous things such as uninstalling both reinstalling etc etc. Whenever I tried to open up lazer it shows that the game is running but it doesn't display at all, but im able to see it through task manager.

**Comment by Axilotl17** [comment]
what operating system are you on? can you send logs? 


log folders:
    `%AppData%/osu/logs` on Windows
    `~/.local/share/osu/logs` on Linux
    `~/Library/Application Support/osu/logs` on macOS

**Comment by Thatcrazytaco222** [comment]
[input.log](https://github.com/ppy/osu/files/11036447/input.log)
[network.log](https://github.com/ppy/osu/files/11036451/network.log)
[performance.log](https://github.com/ppy/osu/files/11036453/performance.log)
[runtime.log](https://github.com/ppy/osu/files/11036455/runtime.log)
[updater.log](https://github.com/ppy/osu/files/11036456/updater.log)

**Comment by peppy** [comment]
Can you try hitting alt-enter one or more times? It looks like everything is working as expected, so my only thought is that your window is stuck off-screen somewhere.

**Reply by Thatcrazytaco222** [reply]
The issue is it doesn't appear as a window whatsoever, but it does pull resources when i'm checking through task manager
